src/StatusQ/Controls/StatusBanner.qml

```diff
@@ -34,14 +48,46 @@ Column {
             verticalAlignment: Text.AlignVCenter
             font.pixelSize: statusBanner.textPixels
             text: statusBanner.statusText
-            color: statusBanner.fontColor
+            color: d.fontColor
         }
     }
 
     Rectangle {
         id: bottomDiv
-        color: statusBanner.bordersColor
+        color: d.bordersColor
         height: 1
         width: parent.width
     }
+
+    // Behavior
+    states: [
+        State {
+            name: "Info"
```

Comment:
Great @noeliaSD, thanks! since we set the `when` property then `name` is not needed, or is it used elsewhere I missed?